src/Icons/src/IconRenderer.php

```diff
@@ -36,11 +42,28 @@ public function __construct(
      */
     public function renderIcon(string $name, array $attributes = []): string
     {
-        $name = $this->iconAliases[$name] ?? $name;
+        $iconName = $this->iconAliases[$name] ?? $name;
+
+        $icon = $this->registry->get($iconName);
+
+        if (false !== $colonPosition = strpos($name, ':')) {
+            $prefix = substr($name, 0, $colonPosition);
+            $iconSetAttributes = $this->iconSetsAttributes[$prefix] ?? [];
+        }
+
+        $icon = $icon->withAttributes(
+            array_replace(
+                $this->defaultIconAttributes,
+                $iconSetAttributes ?? [],
+                $attributes,
+            )
```

Comment:
Not possible to use `+` instead?

Reply:
The order must be kept so i dont think so..  :|

Reply:
Ah? the order matters?

Reply:
Well i just spent an hour, persuaded there were some case it differed, due to the "null"  / "false" .....

..... and no surprise: i was wrong 😮‍💨 

i'll update tomorrow

Reply:
Done, and with this much better tests !  (in the end i chose a ... instead of a +  :)

Reply:
I'll let you check and LGTM :)

Reply:
> Done, and with this much better tests ! (in the end i chose a ... instead of a + :)

Yeah, I find it more readable